sdk/src/main/java/com/silanis/esl/sdk/examples/BasicPackageCreationExample.java

```diff
@@ -47,6 +48,7 @@ public void execute() {
         email2 = "CapitalLetters@email.com";
 
         DocumentPackage superDuperPackage = newPackageNamed(getPackageName())
+                .withTimezoneId(PACKAGE_TIMEZONE_ID)
```

Comment:
I can't understand why a package should have a timezone. Is not the time zone necessary for the user? Please link the Acceptance Criteria of the relevant ticket.

Reply:
Please see above comment for explanation.